addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -79,10 +79,19 @@ function factory(dependencies) {
             const value = req.term;
             const escapedValue = owl.utils.escape(value);
             this.update({ addingChannelValue: value });
+            const domain = [
+                ['channel_type', '=', 'channel'],
+                ['public', '!=', 'private'],
```

Comment:
I'm wondering if we really need this line about `public`, and probably not. Feels like we could/should rely on ACL here, and just return all channels the current user has the rights to see.

As it is now, private channels of which current user is member won't show here, even though that's what we would like.